src/main/java/com/google/cloud/tools/appengine/api/devserver/RunConfiguration.java

```diff
@@ -81,4 +81,9 @@
   File getDatastorePath();
 
   Map<String, String> getEnvironment();
+
+  /**
+   * Valid only for dev_appserver1.
```

Comment:
This suggests an abstraction failure. It really shouldn't care 1 or 2. That's why the interface exists. And what does it do for dev_appserver2? Throw an UnsupportedOperationException?

Also, nees JavaDoc